module-system/module-implementations/sov-evm/src/evm/mod.rs

```diff
@@ -1,4 +1,54 @@
-mod db;
+use sov_state::Prefix;
+
+mod conversions;
+pub(crate) mod db;
+mod db_commit;
+mod db_init;
 mod executor;
 #[cfg(test)]
 mod tests;
+
+pub(crate) type Address = [u8; 20];
+pub(crate) type SovU256 = [u8; 32];
+
+// Stores information about an EVM account
+#[derive(borsh::BorshDeserialize, borsh::BorshSerialize, Debug, PartialEq, Clone, Default)]
+pub(crate) struct AccountInfo {
+    pub(crate) balance: SovU256,
+    pub(crate) code_hash: SovU256,
+    // TODO: `code` can be a huge chunk of data. We can use `StateValue` and lazy load it only when needed.
+    // https://github.com/Sovereign-Labs/sovereign-sdk/issues/425
+    pub(crate) code: Vec<u8>,
+    pub(crate) nonce: u64,
+}
+
+/// Stores information about an EVM account and a corresponding account state.
+#[derive(borsh::BorshDeserialize, borsh::BorshSerialize, Debug, PartialEq, Clone)]
+pub(crate) struct DbAccount {
+    pub(crate) info: AccountInfo,
+    pub(crate) storage: sov_state::StateMap<SovU256, SovU256>,
```

Comment:
(question): It looks like we're saving the `StateMap` into state (which in practice means serializing and storing the prefix for the inner Map). Do we need to do that? Or can we reconstruct the prefix from the Account address whenever we need it?

Reply:
We could do that, I did it this way for 2 reasons:
- tokens in bank already work this way (but we could change that too if we decide to)
- here is how we create the account storage prefix: 
```Rust
let mut prefix = parent_prefix.as_aligned_vec().clone().into_inner();
prefix.extend_from_slice(&address);
```
If we reconstruct the prefix when we need it would incur some additional work on every state access.

Reply:
Interesting. I think reconstructing it once (when we load the account) will likely still be a lot cheaper than reading from storage, since storage data gets hashed. But that's an optimization - no need to worry about it for now